### DISCUSSION TITLE
Add Z-index Override for Time Series Panel

### DISCUSSION THREAD

**Discussion by kthompsonpros** [grafana/grafana#37986] [Dashboards, Panels & Visualizations]
Hello, 

In regards to the "Time series panel visualization is going to replace the Graph panel visualization in a future release", the Z-index override feature missing from the Time Series panel is not ideal. 

Our data source is Prometheus, and the built in data transformations will not work for our use case because of how dynamic the panels are based on the dashboard variables and metric label values being used in the legends; I did not find and regex support in transformations, just static series names. 
For example, we have a Graph panel that shows the response time and traffic per customer, and the customer ID is retrieved via a metric label which we then display all the customer IDs as response time and traffic as series. We have the traffic displayed in the background as bars using the Z-index override. 

Renaming the queries, or deleting and creating a new is the only way I have found to re-order the display, but that is not definite rule telling the series what position to use and more having faith it will remain in that order. It would be extremely useful if the Z-index override for the new Time Series panel.

Thank you, 

Keith T.

**Comment by leeoniya** [comment]
related on our radar: https://github.com/grafana/grafana/issues/37904

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
